dataRetriever/resolvers/topicResolverSender/topicResolverSender_test.go

```diff
@@ -82,6 +85,18 @@ func TestNewTopicResolverSender_NilOutputAntiflooderShouldErr(t *testing.T) {
 	assert.Equal(t, dataRetriever.ErrNilAntifloodHandler, err)
 }
 
+func TestNewTopicResolverSender_InvalidNumberOfPeersShouldErr(t *testing.T) {
+	t.Parallel()
+
+	arg := createMockArgTopicResolverSender()
+	arg.NumIntraShardPeers = 1
```

Comment:
How if NumIntraShardPeers=2 and NumCrossShardPeers=0 or vice versa?

Reply:
this is a valid configuration for the resolver (for our system we certainly we do not want 0 on intrashard peers but that is the job of the factory). The resolver doesn't care much of the desired configuration. Added a test with values 0 and 2